src/System.Text.Json/src/System/Text/Json/Serialization/ReadStack.cs

```diff
@@ -10,7 +11,9 @@ namespace System.Text.Json.Serialization
     [DebuggerDisplay("Current: ClassType.{Current.JsonClassInfo.ClassType}, {Current.JsonClassInfo.Type.Name}")]
     internal struct ReadStack
     {
-        // A fields is used instead of a property to avoid value semantics.
+        private static readonly char[] SpecialCharacters = { '.', ' ', '\'', '/', '"', '[', ']', '(', ')', '\t', '\n', '\r', '\f', '\b', '\\', '\u0085', '\u2028', '\u2029' };
```

Comment:
Is this the same as Json.NET?

Reply:
Yes verbatim pending review. cc @ahsonkhan

Reply:
`'\u0085', '\u2028', '\u2029'` might be concerning, but I am not sure if it this issue applies here: https://github.com/dotnet/corefx/issues/33321

cc @GrabYourPitchforks, @bartonjs